documentation/inventory_files.md

```diff
@@ -468,6 +468,145 @@ plugins:
         message: Enter your Vault password
 ```
 
+### Share inventory between projects
+
+Because projects are used to run a specific workflow on a list of targets, it's
+common to have multiple projects that each run a different workflow on the same
+list of targets. If you have multiple projects that all use the same inventory,
+it can be useful to use a shared inventory file so you don't need to maintain
+your inventory in each project individually.
+
+**Using the command line**
```

Comment:
```suggestion
#### Using the command line
```
Let's use H4s so we can link directly to these sections if needed. They're essentially identical to bold anyway